base/third_party/go-wrapper/go-wrapper

```diff
@@ -89,11 +89,14 @@ case "$cmd" in
 		if [ -f "./source-context.json" ]; then
 			service="${GAE_SERVICE:-default}"
 			version="${GAE_VERSION:-default}"
-			set -x; exec go-cloud-debug \
+			set +e; set -x; go-cloud-debug \
 				-sourcecontext ./source-context.json \
 				-appmodule "${service}" \
 				-appversion "${version}" \
 				-- "$(which $goBin)" "$@"
```

Comment:
Just to ensure the `"$?"` behavior is okay, move the "$(which $goBin)" behavior to a variable before this line.